lib/libmuser.c

```diff
@@ -77,16 +77,21 @@ typedef struct {
     _Static_assert(sizeof(s) - offsetof(s, m) == sizeof(t), \
         #t " " #m " must be last member in " #s)
 
+struct pci {
```

Comment:
Should this be `lm_ctx_pci`?

Reply:
This struct is internal to the library, so there's no problem of namespace clash. Do we want to prefix internal structs as well?